deb/common/docker-ce.docker.service

```diff
@@ -0,0 +1 @@
+engine/contrib/init/systemd/docker.service
```

Comment:
How does this work? 🤔 Can the file contain the name of "actual file to look for"? Or is this just a "placeholder"? (trying to understand what this does)

Reply:
Oh sorry, this is a symlink!  We could probably replace the other init scripts with symlinks too (if they're unchanged from what's in `moby/moby`).

Reply:
Oh! LOL

Didn't expect it to show up as a regular file in github's UI 🤦‍♂️😂

Reply:
Yeahhhhh, I really wish symlinks were more obviously labelled in diffs, but at least it shows something! :smile: